src/averageAges.js

```diff
@@ -15,11 +32,24 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  const age = people
+    .filter(x => {
+      if (century) {
```

Comment:
replace `if ()` statement with &&, || or ?:

Reply:
Danylo , mentor, said that it's better to use 'if ()' statement

Reply:
if's task condition )